Documentation/en-us/README.md

```diff
@@ -29,7 +29,7 @@ testing, consider reading them in the order they're introduced below:
   Read this to learn more about testing code that uses the AppKit and UIKit frameworks.
 - **[Testing with test doubles](TestUsingTestDoubles.md)**:
   Read this to learn what test doubles are and how to use them.
-- **[Reducing Test Boilerplate with Shared Assertions](SharedExamples.md)**:
+- **[Reducing Test Boilerplate with Behaviors and Shared Assertions](SharedExamples.md)**:
```

Comment:
I think we can define "Shared Assertions" as a concept including both `Behavior` and `sharedExample` implementation, so I'd like to leave this as is.